profiler/hotapp/Dockerfile

```diff
@@ -1,12 +1,10 @@
 FROM golang:alpine
 
-WORKDIR /go/src/hotapp
-COPY *.go .
-
-RUN apk update \
-    && apk add --no-cache git \
-    && go get -d ./... \
-    && apk del git
```

Comment:
Any reason to remvoe the lines besides the `go get -d ...`? These would pull in newer dependency versions when building.

Reply:
`go get -d` was deprecated in favor of go install. 
The images are not refreshed automatically and have to be manually updated, so this file runs very infrequently - I doubt this will give us much value.